src/containers/EditCard/EditCard.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import './EditCard.scss';
+import { connect } from 'react-redux';
+import { editCard } from '../../actions';
+
+class EditCard extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Doesn't matter but just FYI you can destructure props in this way too:
```
this.state = { id, title, body, priority_id, status_id, created_by, assigned_to} = props.card;
```
Per conversation with Ed, DON'T DO THIS (but it works, and subject to debate whether it's "wrong" or not)